invenio_app_ils/circulation/api.py

```diff
@@ -62,6 +63,13 @@ def create_loan(params):
         document_pid = circulation_document_retriever(params["item_pid"])
         if document_pid:
             params["document_pid"] = document_pid
+
+    if should_force_checkout:
```

Comment:
May I suggest a small change if you don't mind?
```
if should_force_checkout:
    item = Item.get_record_by_pid(params['item_pid'])
    if item.status != "LOANABLE":
        circulation_item_patcher(item, ...)
```
This will avoid to patch it if it is already LOANABLE (I wouldn't trust the UI)